cmd/cosign/cli/verify/verify.go

```diff
@@ -500,3 +501,20 @@ func keylessVerification(keyRef string, sk bool) bool {
 	}
 	return true
 }
+
+func keylessVerificationWithSCTEnabled(ignoreSCT bool, keyRef string, sk bool, certChain string) bool {
+	rootEnv := env.Getenv(env.VariableSigstoreRootFile)
+	if keyRef != "" {
+		return false
+	}
+	if sk {
+		return false
+	}
+	if ignoreSCT && (certChain != "" || rootEnv != "") {
```

Comment:
This logic can be simplified, you can remove this conditional since only the value of `ignoreSCT` matters.

Reply:
Agree i will remove the condition check for certChian and root ENV will check only for ignoreSCT in the mentioned condition.